Doc/library/os.rst

```diff
@@ -3465,6 +3465,10 @@ written in Python, such as a mail server's external command delivery program.
    :c:data:`POSIX_SPAWN_SETSCHEDPARAM` and :c:data:`POSIX_SPAWN_SETSCHEDULER`
    flags.
 
+  If the value of *use_vfork* is true, the child will be created using
+  :c:func:`vfork` instead of :c:func:`fork`. This corresponds to the
+  GNU specific flag :c:data:`POSIX_SPAWN_USEVFORK`.
+
```

Comment:
Oh, "Availability: ..." is missing: this function is not available on Windows, right?

I would suggest to document in "Availability: ..." that the flag is only available on Linux.